packages/utilities/psammead-storybook-helpers/index.test.jsx

```diff
@@ -0,0 +1,113 @@
+import { boolean, select, text } from '@storybook/addon-knobs';
```

Comment:
This file should be inside src

Reply:
I created `index.test.jsx` in the `packages/utilities/psammead-storybook-helpers` and not `src` level following the implementation in the other packages under `packages/utilities`.
Is the implementation here supposed to be different?

Reply:
Those utilities had to follow that pattern as the things it exported weren't inherently testable, as this exports functions I think it should follow the structure of a component

package.json (with 'main' pointing at dist)
src
--index.jsx
--index.test.jsx

Reply:
Understood. Thanks once again.
I'll make the change as advised.